scripts/build_automation/summarize-test-results.swift

```diff
@@ -24,7 +24,7 @@
 //   swift sh edit summarize-test-results.swift
 
 import Foundation
-import XCResultKit // @cobbal == dev
+import XCResultKit // @davidahouse
```

Comment:
who is this?

Reply:
It points swift-sh to the dependency at https://github.com/davidahouse/XCResultKit